doc/modules/ensemble.rst

```diff
@@ -11,12 +11,13 @@ base estimators built with a given learning algorithm in order to improve
 generalizability / robustness over a single estimator.
 
 Two very famous examples of ensemble methods are `gradient-boosted trees
-<gradient_boosting>`_ and `random forests <forest>`_.
+<gradient-boosting>`_ and
+`random forests <random-forests-and-other-randomized-tree-ensembles>`_.
```

Comment:
Same comment as below, cross-references are fine, they are only missing the `:ref:` and have an extra underscore at the end.

Reply:
Much better indeed :smile: